Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md

```diff
@@ -0,0 +1,373 @@
+benefits.
+
+_Background:_
+  For data retrieval ArangoSearch Views follow the concept of
+  "eventually-consistent", i.e. eventually all the data in ArangoDB will be
+  matched by corresponding query expressions.
+  The concept of ArangoSearch View "commit" operation is introduced to
+  control the upper-bound on the time until document addition/removals are
+  actually reflected by corresponding query expressions.
+  Once a "commit" operation is complete all documents added/removed prior to
+  the start of the "commit" operation will be reflected by queries invoked in
+  subsequent ArangoDB transactions, in-progress ArangoDB transactions will
+  still continue to return a repeatable-read state.
+
+@RESTBODYPARAM{consolidationIntervalMsec,integer,optional,int64}
+Wait at least this many milliseconds between applying 'consolidationPolicy' to
+consolidate View data store and possibly release space on the filesystem
+(default: 1000, to disable use: 0).
+For the case where there are a lot of data modification operations, a higher
+value could potentially have the data store consume more space and file handles.
+For the case where there are a few data modification operations, a lower value
+will impact performance due to no segment candidates available for
+consolidation.
+
+_Background:_
+  For data modification ArangoSearch Views follow the concept of a
+  "versioned data store". Thus old versions of data may be removed once there
+  are no longer any users of the old data. The frequency of the cleanup and
+  compaction operations are governed by 'consolidationIntervalMsec' and the
+  candidates for compaction are selected via 'consolidationPolicy'.
+
+@RESTBODYPARAM{consolidationPolicy,object,optional,}
+The consolidation policy to apply for selecting which segments should be merged
+(default: {}).
+
+_Background:_
+  With each ArangoDB transaction that inserts documents one or more
+  ArangoSearch internal segments gets created.
+  Similarly for removed documents the segments that contain such documents
+  will have these documents marked as 'deleted'.
+  Over time this approach causes a lot of small and sparse segments to be
+  created.
+  A "consolidation" operation selects one or more segments and copies all of
+  their valid documents into a single new segment, thereby allowing the
+  search algorithm to perform more optimally and for extra file handles to be
+  released once old segments are no longer used.
+
+Sub-properties:
+  - `type` (string, _optional_):
+    The segment candidates for the "consolidation" operation are selected based
+    upon several possible configurable formulas as defined by their types.
+    The currently supported types are:
+    - `"tier"` (default): consolidate based on segment byte size and live
+      document count as dictated by the customization attributes. If this type
+      is used, then below `segments*` and `minScore` properties are available.
+    - `"bytes_accum"`: consolidate if and only if
```

Comment:
AFAIK we wanted to mark bytes_accum as deprecated. Let's mention this or even remove it here.

Reply:
Removing, so that people don't start using inverted indexes with a deprecated option.